tools/spelling/check_spelling_pedantic.py

```diff
@@ -27,7 +27,8 @@ def cmp(x, y):
     return (x > y) - (x < y)
 
 
-TOOLS_DIR = os.path.dirname(os.path.realpath(__file__))
+CURR_DIR = os.path.dirname(os.path.realpath(__file__))
+TOOLS_DIR = os.path.dirname(CURR_DIR)
```

Comment:
Let's get rid of TOOLS_DIR altogether. I think the only remaining use is for the `.aspell.en.pws` file that's recreated on each run anyway.

Reply:
Yep, makes sense. Addressed in https://github.com/envoyproxy/envoy/pull/10065/commits/977d6809e07dff51be13c8ddecb9461349a7a629